stories/storyhelper.tsx

```diff
@@ -1,9 +1,35 @@
-import React, { ReactChildren, ReactNode } from 'react';
+import React, { ReactNode } from 'react'
+import { GlobalStyle } from '../theme/globalStyles'
+import { ThemeProvider } from 'styled-components'
+import { theme } from '../theme'
 
 export const Spacer = ({ children }: { children: ReactNode }) => (
-  <div style={{
-    marginBottom: '20px',
-  }}>
+  <div
+    style={{
+      marginBottom: '20px',
+    }}
+  >
     {children}
   </div>
-);
\ No newline at end of file
+)
+
+export const Container = ({ children }: { children: ReactNode }) => (
+  <div style={{ width: '224px', margin: '0 auto' }}>
+    <ThemeProvider theme={theme}>
+      <GlobalStyle />
+      {children}
+    </ThemeProvider>
+  </div>
+)
+
+export const HR = () => (
+  <hr
```

Comment:
@jfcurran this is the divider, you could also use it in the button components if you want